apps/benchmarks_keeper/src/config.rs

```diff
@@ -0,0 +1,21 @@
+use alloy::primitives::Address;
+use config::{Config as ConfigTrait, ConfigError, Environment, File};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct Config {
+    pub rpc_url: String,
+    pub contract_address: Address,
+    pub hermes_url: String,
+}
+
+impl Config {
+    pub fn new() -> Result<Self, ConfigError> {
+        let config = ConfigTrait::builder()
```

Comment:
I don't really understand the point of the `config` crate. Why would you want to implicitly merge a config from multiple different sources? I would much rather prefer that every value comes from a single well-defined, well-documented source, like a config file. It would also be nice to be able to specify the config file location.